packages/frontend/src/models/RecipeModelInfo.ts

```diff
@@ -18,6 +18,7 @@
 
 import type { ModelInfo } from '@shared/src/models/IModelInfo';
 
+// todo :remove
```

Comment:
Can't we remove it now ?

Reply:
Sadly no, there are some weird dependency on it yet that I can't remove

https://github.com/containers/podman-desktop-extension-ai-lab/blob/df409dd5a854bfc288768c0e37459a2da157ceb5/packages/frontend/src/lib/table/model/ModelColumnRecipeSelection.svelte#L6

Moreover there are some work being done in the Table component, which will require some work to be done to fix it. I would prefer keep the todo remove as this need to get addressed when the Table component will be updated